src/CMakeLists.txt

```diff
@@ -689,7 +689,7 @@ if(WITH_FUSE)
     client/fuse_ll.cc)
   add_executable(ceph-fuse ${ceph_fuse_srcs})
   target_link_libraries(ceph-fuse ${FUSE_LIBRARIES}
-    ${GSSAPI_LIBRARIES} client ceph-common global-static)
+    ${EXTRALIBS} ${GSSAPI_LIBRARIES} client ceph-common global-static)
```

Comment:
shall we move `${EXTRALIBS}` after `ceph-common` or even after `global-static`? i am afraid that some compilation units in them might reference `clock_gettime()` and other symbols defined by `${EXTRALIBS}`.

Reply:
That's ok, i add the item just refer to the use of it in src/CMakeLists.txt and build it successfully.